Hw11/Hw11.cpp

```diff
@@ -0,0 +1,103 @@
+#include <iostream>
+using namespace std;
+
+bool lS2D(const int arr[][3], int rows, int cols, int target) {
```

Comment:
`bool find(int arr_2d[ROWS][COLUMNS], int toFind)`
declaration should be like this

Reply:
what is 3??